src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -323,6 +323,26 @@ var getCurrentUrlPath = () => {
   return `${DOCUMENTATION_OPTIONS.pagename}.html`;
 };
 
+/**
+ * Allow user to dismiss the warning banner about the docs version being dev / old.
+ * We store the dismissal date and version, to give us flexibility about making the
+ * dismissal last for longer than one browser session, if we decide to do that.
+ *
+ * @param {event} event the event that trigger the check
+ */
+async function DismissBannerAndStorePref(event) {
+  const banner = document.querySelector("#bd-header-version-warning");
+  banner.remove();
+  let version = DOCUMENTATION_OPTIONS.VERSION;
+  let now = new Date();
+  let banner_pref = JSON.parse(localStorage.getItem("pst_banner_pref") || "{}");
```

Comment:
```suggestion
  const version = DOCUMENTATION_OPTIONS.VERSION;
  const now = new Date();
  const banner_pref = JSON.parse(localStorage.getItem("pst_banner_pref") || "{}");
```

you were right about `const`, I often use `let` out of fewer-characters laziness